core/src/main/scala/org/http4s/middleware/authentication/DigestUtil.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s.middleware.authentication
```

Comment:
This is a new package, right?  I think we should stick this in `org.http4s.internal`.

Reply:
updated